server/forms/threats.js

```diff
@@ -183,3 +183,74 @@ exports.validate = {
     }
   }
 }
+
+exports.prepareCsv = async function (api, record, csv) {
+  // define all used fields, so the first record can output proper header
+  csv = {
+    ...csv,
+    id: record.id,
+    categoryThreatsBG: '',
+    categoryThreatsEN: '',
+    speciesThreats: '',
+    countThreats: '',
+    estimateThreatsBG: '',
+    estimateThreatsEN: '',
+    deadSpecies: '',
+    deadSpeciesCount: '',
+    aliveAnimal: 'N',
+    aliveAnimalCount: '',
+    poisonBaits: 'N',
+    poisonBaitsCount: ''
+  }
+  switch (record.primaryType) {
+    case formsConfig.threatsPrimaryTypes.poison.id:
+      csv = {
+        ...csv,
+        categoryThreatsBG: 'Тровене',
+        categoryThreatsEN: 'Poisoning'
+      }
+
+      switch (record.poisonedType) {
+        case formsConfig.threatsPoisonedType.alive.id:
+          csv = {
+            ...csv,
+            aliveAnimal: 'Y',
+            aliveAnimalCount: record.count
+          }
+          break
+        case formsConfig.threatsPoisonedType.bait.id:
+          csv = {
+            ...csv,
+            poisonBaits: 'Y',
+            poisonBaitsCount: record.count
+          }
+          break
+        case formsConfig.threatsPoisonedType.dead.id:
+          csv = {
+            ...csv,
+            aliveAnimal: 'N',
```

Comment:
This is not necessary since it is 'N' in definition above.
```suggestion

```

Reply:
To be explicit for the case where animal is dead in case default gets changed to something like empty string